src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerReviewEvidenceNoteTests.cs

```diff
@@ -0,0 +1,334 @@
+            //arrange
+            var httpContext = new HttpContextMocker();
+            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
+            AddModelError();
+            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
+            httpContext.AttachToController(ManageEvidenceController);
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as ViewResult;
+
+            //assert
+            result.Model.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenAModelIsValid_ShouldRedirectToDownloadEvidenceNoteAction()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+            var model = GetValidModel();
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as RedirectToRouteResult;
+
+            // assert
+            result.RouteValues["action"].Should().Be("DownloadEvidenceNote");
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenInvalidModel_ShouldRedirectToReviewEvidenceNoteAction()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
+            AddModelError();
+            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
+            httpContext.AttachToController(ManageEvidenceController);
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as ViewResult;
+
+            // assert
+            result.ViewName.Should().Be("ReviewEvidenceNote");
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenRequestIsCreated_SessionShouldNotBeNull()
+        {
+            //arrange
+            var model = GetValidModel();
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+
+            //act
+            await ManageEvidenceController.ReviewEvidenceNote(model);
+
+            // assert
+            ManageEvidenceController.Session.Should().NotBeNull();
+            ManageEvidenceController.Session.SessionID.Should().BeNullOrEmpty();
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenInvalidModel_SessionShouldBeEmpty()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
+            AddModelError();
+
```

Comment:
Not sure of the intention of this test. The ReviewEvidenceNote action doesnt use the session anywhere. It it is copy and paste it can be removed.